### DISCUSSION TITLE
Change Label Position on <Switch />

### DISCUSSION THREAD

**Discussion by JaredDeanFP** [adobe/react-spectrum#6047] [Q&A] [answered]
Is there a simple way to change the Switch's label from being on the right to being on the left?

What its doing:
<img width="316" alt="image" src="https://github.com/adobe/react-spectrum/assets/109235744/38b87190-1075-47bb-b737-e45a2bc8562b">

What I want:
<img width="347" alt="image" src="https://github.com/adobe/react-spectrum/assets/109235744/0fdbd9b4-e12f-4979-aaa0-dd74f5d9b281">

**Comment by reidbarber** [comment, marked as answer]
For React Aria Component's Switch, you control what gets rendered as the visual indicator. In the [docs example](https://react-spectrum.adobe.com/react-aria/Switch.html#example), you would move the indicator after the label:

```jsx
<Switch>
  Low power mode
  <div className="indicator" />
</Switch>
```

If you want to support both cases, you could make a prop on your wrapper that flips their positioning in render, or with CSS (i.e. `flex-direction: row-reverse`, or CSS grid)

**Reply by JaredDeanFP** [reply]
Ah, now I feel foolish 😆 thank you
